eventbridge-cloudevents-transformer-cdk-typescript/src/test/eb-lambda-cloudevents.test.ts

```diff
@@ -0,0 +1,17 @@
+// import * as cdk from 'aws-cdk-lib';
+// import { Template } from 'aws-cdk-lib/assertions';
+// import * as EbLambdaCloudevents from '../lib/eb-lambda-cloudevents-stack';
+
+// example test. To run these tests, uncomment this file along with the
+// example resource in lib/eb-lambda-cloudevents-stack.ts
+test('SQS Queue Created', () => {
```

Comment:
Think we can remove this file @embano1 ?

Reply:
Yeah, everyone had it in so I though #YOLO :D